SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -696,23 +702,25 @@ void USpatialSender::SendOnEntityCreationRPC(UObject* TargetObject, UFunction* F
 #endif // !UE_BUILD_SHIPPING
 }
 
-void USpatialSender::SendCrossServerRPC(UObject* TargetObject, UFunction* Function, const SpatialGDK::RPCPayload& Payload,
+void USpatialSender::SendCrossServerRPC(UObject* TargetObject, UFunction* Function, SpatialGDK::RPCPayload Payload,
```

Comment:
Why is this now not const ref?

Reply:
It now needs to modify it if event tracing is enabled, I guess it's possible to write this code to copy only if event tracing is enabled